src/Layout/index.js

```diff
@@ -93,6 +96,28 @@ function Layout(props) {
             {children}
           </div>
           {!noFooter && <Footer />}
+          <CookieConsent
```

Comment:
Can you move this to its own component? It should be able to be overridden (props passed in) but I want to keep `Layout` a little more clean.

Reply:
Haha. I just moved it out and then saw your review.

Reply:
I agree, it shouldn't be clogging up `Layout`

Reply:
The props can be overriden, but I don't want to clog up the theme folder in lips/docs. Therefore, I'm not going to make the overriding come from a component, because then regardless of if things are any different, both repos would have to add a CookieBanner component folder/file to avoid a webpack error. What I'm going to do instead is make this overridable from the docusaurus.config.js file as a custom field. The only downside of this is the children prop can't be overriden, but I think that's fine. If we ever did need to do that (which I think would be unlikely), we could make the individual parts of children overridable and then pass those in